sunpy/net/helio/hec.py

```diff
@@ -152,7 +156,7 @@ def search(self, *args, **kwargs):
             table = str.encode(table)
         start_time = qrdict['Time'].start
         end_time = qrdict['Time'].end
-        max_records = qrdict.get('max_records', 10)
+        max_records = qrdict.get('max_records', 500)
```

Comment:
I thought it was going to be 50 - makes no real odds though

Reply:
This way, you can do a few days or so without problem.